pages/clubs-toolkit.jsx

```diff
@@ -184,7 +186,7 @@ var ClubsToolkit = React.createClass({
               <p>Tell the world what you and your club are doing! Here are ways to show off the hard work that’s put into teaching and learning the web.</p>
               <ToolkitList>
                 <ToolkitListItem header="Social Media.">
-                  Share your club activities online. You might even consider creating a new hashtag just for your club so the learners can keep track of all that happens in one place. Write a few sample tweets and posts that your mentors and learners can share on their networks as well. Be sure to share them with us too using <a href="https://twitter.com/search?q=%23teachtheweb">#teachtheweb</a> and <a href="https://twitter.com/webmaker">@Webmaker</a>.
+                  Share your club activities online. You might even consider creating a new hashtag just for your club so the learners can keep track of all that happens in one place. Write a few sample tweets and posts that your mentors and learners can share on their networks as well. Be sure to share them with us too using <a href="https://twitter.com/search?q=%23teachtheweb">#teachtheweb</a> and <a href={config.TWITTER_LINK}>@Webmaker</a>.
```

Comment:
Hmm it looks like the actual text of the link still says `@Webmaker`, maybe make a `config.TWITTER_NAME` too?